Install fails

# Goal

The installation fails because `flatbuffers` cannot be found, this needs to be fixed for installation to complete on a fresh install

# Additional information

Trace:

```
In file included from /home/pi/PosturePerfection/src/inference_core.h:25,
                 from /home/pi/PosturePerfection/src/post_processor.h:29,
                 from /home/pi/PosturePerfection/src/iir.cpp:21:
/home/pi/tensorflow_src/tensorflow/lite/interpreter_builder.h:24:10: fatal error: flatbuffers/flatbuffers.h: No such file or directory
 #include "flatbuffers/flatbuffers.h"  // from @flatbuffers
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [src/CMakeFiles/PosturePerfection_static.dir/build.make:82: src/CMakeFiles/PosturePerfection_static.dir/iir.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:1199: src/CMakeFiles/PosturePerfection_static.dir/all] Error 2
make: *** [Makefile:182: all] Error 2
```

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -26,6 +26,7 @@ add_subdirectory(
 
   
 add_library(PosturePerfection_static STATIC ${LIBSRC})
+target_include_directories(PosturePerfection_static PUBLIC "${CMAKE_CURRENT_BINARY_DIR}/../flatbuffers/include")
 
 add_executable(PosturePerfection ${LIBSRC} main.cpp)
 target_link_libraries(PosturePerfection tensorflow-lite ${CMAKE_DL_LIBS})

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -18,14 +18,15 @@ endif()
 # Ensure all necessary libraries are included
 include_directories("${TENSORFLOW_SOURCE_DIR}")
 include_directories("${TENSORFLOW_SOURCE_DIR}/tensorflow/lite")
-include_directories("${TENSORFLOW_SOURCE_DIR}/../tflite_build/flatbuffers/include/")
 
 # Add and build TFL
 add_subdirectory(
   "${TENSORFLOW_SOURCE_DIR}/tensorflow/lite"
   "${CMAKE_CURRENT_BINARY_DIR}/tensorflow-lite" EXCLUDE_FROM_ALL)
 
+  
 add_library(PosturePerfection_static STATIC ${LIBSRC})
 
 add_executable(PosturePerfection ${LIBSRC} main.cpp)
 target_link_libraries(PosturePerfection tensorflow-lite ${CMAKE_DL_LIBS})
+target_include_directories(PosturePerfection PRIVATE "${CMAKE_CURRENT_BINARY_DIR}/../flatbuffers/include")

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -2,6 +2,7 @@ set(CMAKE_CXX_STANDARD 11)
 cmake_minimum_required(VERSION 3.16) # Required by TFL
 project(PosturePerfection VERSION 0.1.0)
 
+set(CMAKE_CXX_LINK_FLAGS "${CMAKE_CXX_LINK_FLAGS} -latomic")
 
 set(LIBSRC
   iir.cpp

--- a/test/CMakeLists.txt
+++ b/test/CMakeLists.txt
@@ -13,8 +13,6 @@ endif()
 # Ensure all necessary libraries are included
 include_directories("${TENSORFLOW_SOURCE_DIR}")
 include_directories("${TENSORFLOW_SOURCE_DIR}/tensorflow/lite")
-include_directories("${TENSORFLOW_SOURCE_DIR}/../tflite_build/flatbuffers/include/")
-
 
 include_directories ("${TEST_SOURCE_DIR}/../src"
                      ${Boost_INCLUDE_DIRS})

